doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1936,7 +1936,7 @@ Abbreviations
 
    .. rocqtop:: all
 
-      Compute (Plus1 3).
+      Eval compute in (Plus1 3).
```

Comment:
I was also unable to reproduce that one.
Also note that there is a small change of semantics as `Compute` is a misleading alias for `Eval vm_compute in`, although we probably don't care in this context.
(my opinion remains that we should deprecate and remove that misleading `Compute` command but we couldn't find an agreement on that)

Reply:
Precisely. The change of semantics was the reason for making this change. For the `warn` above and this change, I obtained a warning about Rocq having been configured without `vm_compute`, and the code falling back to `compute`. However, I cannot reproduce the warning behavior anymore, so I guess this must have been an inconsistent compilation problem... Thus, I agree with dropping the commit with these two changes.